### PR TITLE
Visitor can see crime teasers

### PR DESCRIPTION
**Pull request by ghost**
[PT](https://www.pivotaltracker.com/story/show/175012626)

## This PR contains:

- Acceptance test for Visitor can see crime teasers feature.
- TeaserList component.
- DisplayTeaserList unit test.
- Modules file reports.js.
- Crime data json file.

image

### CODE REVIEW

File: cypress/integration/visitorCanSeeCrimeTeasers.feature.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    });
+
+    it("shows second teaser", () => {
+        cy.get("[data-cy=teaser-2]").within(() => {
+            cy.contains("Robbery in Kista");
+            cy.contains("A woman got her cellphone stolen in the Kista subway station.");
+            cy.get("button").should("contain", "Read more").click()
+        });
+    });
+});
```
Comment (tochman) [inline cypress/integration/visitorCanSeeCrimeTeasers.feature.js:24]: Please be consistent in using either 2 or 4 spaces when formatting code.

File: cypress/integration/visitorCanSeeCrimeTeasers.feature.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            cy.contains("A man robbed a student of his wallet.");
+            cy.get("button").should("contain", "Read more").click()
+        });
+    });
+
+    it("shows second teaser", () => {
+        cy.get("[data-cy=teaser-2]").within(() => {
+            cy.contains("Robbery in Kista");
+            cy.contains("A woman got her cellphone stolen in the Kista subway station.");
+            cy.get("button").should("contain", "Read more").click()
```
Comment (tochman) [inline cypress/integration/visitorCanSeeCrimeTeasers.feature.js]: ..and then what?

File: cypress/integration/visitorCanSeeCrimeTeasers.feature.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            cy.contains("Robbery in gothenburg"); 
+            cy.contains("A man robbed a student of his wallet.");
+            cy.get("button").should("contain", "Read more").click()
+        });
+    });
+
+    it("shows second teaser", () => {
+        cy.get("[data-cy=teaser-2]").within(() => {
+            cy.contains("Robbery in Kista");
+            cy.contains("A woman got her cellphone stolen in the Kista subway station.");
```
Comment (tochman) [inline cypress/integration/visitorCanSeeCrimeTeasers.feature.js]: If you are making an assertion, you should use the `.should` syntax.

**Review by tochman — COMMENTED** [review]
Not really ready for prime-time, but it's still marked WIP so that's okay...

File: src/App.jsx [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
-        
-      </div>
-    )
-  }
+import React from 'react'
+import TeaserList from "./components/TeaserList"
+const App = () => {
+  return (
+    <>
+      <h1 data-cy='header'>Teaser List</h1>
```
Comment (emtalen) [inline src/App.jsx:6]: Do you really want it to say Teaser List?

**Review by emtalen — COMMENTED** [review]
I think everything looks really good with the functionality of the feature but I would expect more regarding UX and some styling.

**Pull request merged**